Packs/AWS/Integrations/AWS/AWS.yml

```
@@ -290,6 +320,40 @@ script:
     - description: 'The prefix to be used for log object keys (e.g., "logs/"). Objects will be stored as: targetBucket/targetPrefix/sourceBucket/YYYY-MM-DD-HH-MM-SS-UniqueString.'
       name: target_prefix
       required: false
+
+  - name: aws-s3-bucket-enable-bucket-access-logging-quick-action
+    description: Enables server access logging for the S3 bucket, delivering detailed logs to a target bucket and prefix by running the aws s3api put-bucket-logging command. Successful execution requires the s3:PutBucketLogging permission.
```

```suggestion
    description: Enables server access logging for the S3 bucket, delivering detailed logs to a target bucket and prefix by running the aws s3api put-bucket-logging command.
```
Replace aws s3api put-bucket-logging with the command name in content.